docs/api/navigators/DrawerNavigator.md

```diff
@@ -157,25 +153,22 @@ Usually you define static `navigationOptions` on your screen component. For exam
 ```jsx
 class ProfileScreen extends React.Component {
 
-  static navigationOptions = {
+  static navigationOptions = ({ navigation }) => ({
 
-    title: ({ state }) => `${state.params.name}'s Profile!`,
+    title: `${navigation.state.params.name}'s Profile!`,
 
-    drawer: {
-      icon: (
-        <Image src={require('./my-icon.png')} />
-      ),
-    },
-  };
+    drawerIcon: (
+      <Image src={require('./my-icon.png')} />
+    ),
+  });
   ...
 ```
 
 All `navigationOptions` for the `DrawerNavigator`:
```

Comment:
Ugh, so we need to document every navigation option on the navigator page AND the `Screen-Nav-Options` page? Maybe we can remove the options from the navigators page, especially as there is some overlap like `title`.